### PR TITLE
Feature: Ethernet Network

### PR DESCRIPTION
**Pull request by tmiotto**
Added support to `:bulk_create`

### CODE REVIEW

**Comment by jsmartt** [comment]
CHANGELOG?

File: spec/fixtures/cookbooks/oneview_test/metadata.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@
 maintainer       'test cookbook'
 license          'All rights reserved'
 description      'A test cookbook for the oneview cookbook'
-version          '0.1.0'
+version          '0.1.1'
```
Comment (hdiomede) [inline spec/fixtures/cookbooks/oneview_test/metadata.rb:5]: Do we need to change the version for this cookbook. It's just for testing!

**Comment by hdiomede** [comment]
:+1:

File: spec/fixtures/cookbooks/oneview_test/metadata.rb (hunk 1 above)
Comment (tmiotto) [inline spec/fixtures/cookbooks/oneview_test/metadata.rb:5]: No, we don't. It was just to keep in track with the original cookbook version...

**Pull request merged**